services/frontend-service/src/ui/utils/Links.tsx

```diff
@@ -198,3 +198,15 @@ export const setHideWithoutWarnings = (params: URLSearchParams, newValue: boolea
         params.delete(hideWithoutWarningsParamName);
     }
 };
+
+// functions like these are both store and links, should we be consistent?
```

Comment:
I don't understand this comment.

Reply:
see e.g. `useOpenReleaseDialog` and `useCloseReleaseDialog` in store.tsx doing very similar things -- but it seems they are wrong in store.tsx and should be in links.tsx instead?